app/frontend/src/assets/scss/style.scss

```diff
@@ -28,47 +45,32 @@ a {
   }
 }
 
+// General Button Style Extensions
+.v-btn {
+  span > span {
+    font-weight: bold;
+  }
 
-/* sticky footer */
-html,
-body {
-  height: 100%;
-}
-body {
-  display: flex;
-  flex-direction: column;
-}
-body > .v-content {
-  flex: 1 0 auto;
-}
-
-.BC-Gov-PrimaryButton.v-btn {
-  background-color: #003366;
-  border: none;
-  border-radius: 4px;
-  color: white !important;
-  padding: 12px 32px !important;
-  text-align: center;
-  text-decoration: none;
-}
+  &:hover {
+    opacity: 0.8;
+    span > span {
+      text-decoration: underline;
+    }
+  }
 
-.v-btn.v-btn--disabled {
-  background-color: #eee;
-  color: #777;
-}
+  &.v-btn--disabled {
```

Comment:
I don't know if you need the & here. If this is trying to select  v-btn.v-btn--disabled I think just having the nesting accomplishes that...